src/gpu-compute/GPU.py

```diff
@@ -360,6 +360,9 @@ class GPUCommandProcessor(DmaVirtDevice):
     walker = Param.VegaPagetableWalker(
         VegaPagetableWalker(), "Page table walker"
     )
+    target_user_kernel_id = Param.Int(
```

Comment:
should this be tasks instead of kernels?

Reply:
The target kernel ID referenced here is the kernel number not counting blit kernels. I have been using the word "task" to refer to both blit and non-blit kernels, and the word "kernel" to refer only to non-blit kernels. There's probably less confusing terminology that we could use here. (Actually, this variable specifically refers to "user kernels", which I was using to further emphasize that this counts only non-blit kernels. If you can think of a better phrase than "user kernel", I'm very open to rewording this to avoid confusion.)

Reply:
Probably the only thing to do then is update the comment on the next line to reflect what you said here, thanks for the clarification.